x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.tsx

```diff
@@ -337,14 +426,15 @@ export class MapsAppView extends React.Component {
         showSearchBar={true}
         showFilterBar={true}
         showDatePicker={true}
+        // @ts-ignore
         showSaveQuery={getMapsCapabilities().saveQuery}
-        savedQuery={this.state.savedQuery}
+        savedQuery={this.state.savedQuery as SavedQuery}
```

Comment:
Cast should no longer be needed now that this.state.savedQuery can only be SavedQuery

Reply:
Removed the casting.